app/forms/TransactorTelephoneForm.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.constraints.TelephoneNumberConstraints
+import forms.constraints.utils.ConstraintUtil.ConstraintUtil
+import play.api.data.Form
+import play.api.data.Forms.{single, text}
+
+object TransactorTelephoneForm {
+
+  val telephoneNumberKey = "telephoneNumber"
+
+  val form =
+    Form(
+      single(
+        telephoneNumberKey -> text.verifying(
+          TelephoneNumberConstraints.telephoneNumberEmpty andThen
```

Comment:
These constraints are picking up different error messages from the ones you defined and at least one of them is different from the ones in these constraints.